recipe/meta.yaml

```diff
@@ -172,6 +192,13 @@ test:
     # 2024/02 These tests assert warnings and in PyTorch 2.1.2 the number of warnings increased
     # causing them to fail
     {% set tests_to_skip = tests_to_skip + " or test_pretrained_pos or test_equivalent_behavior_weights" %}
+    # use pip list to manually check that the version is correctly reported
+    # Otherwise downstream package's pip check may fail
+    - pip list
+    - pip list | grep torchvision | grep {{ version }}         # [unix]
+    # 2024/12/12 untested windows version of the grep test above
+    - pip list | findstr torchvision | findstr {{ version }}   # [win]
```

Comment:
Why not just add `m2-grep` as a test dependency on windows?

Reply:
because findstr works for this usecase?

I've used it before in similar cases.
https://github.com/conda-forge/teensy_loader_cli-feedstock/blob/6d67e02351b41e349acb3e75ffb3877052a84bc2/recipe/meta.yaml#L51

Reply:
To be fair, if you write "untested", that doesn't sound make a strong case. For example, I wouldn't know if findstr gives the right error code if matches are (not) found, which is why overall I'd prefer using the same test across all platforms. You can keep it as-is if that's what you want.

OTOH, whether the installation got the right version is very unlikely to diverge across platforms, so even removing the test on windows would be fine IMO. 🤷

Reply:
It’s mostly an opportunity for the future developer to take a look at it. I’m happy to remove it if you think it is for the best. But environment variables are tricky and can behave different on windows and Unix, so I would rather be defensive.

Reply:
ok removed.